src/common/json_helper.h

```diff
@@ -335,6 +337,23 @@ INT_TYPE_SERIALIZATION(int16_t)
 INT_TYPE_SERIALIZATION(int32_t)
 INT_TYPE_SERIALIZATION(int64_t)
 
+template <typename Writer>
+inline void json_encode(Writer &out, dsn::ranger::access_type t)
+{
+    out.Uint64(static_cast<uint64_t>(t));
+}
+
+inline bool json_decode(const JsonObject &in, dsn::ranger::access_type &t)
+{
+    dverify(in.IsUint64());
+    int64_t ans = in.GetUint64();
+    int64_t act_min = static_cast<uint64_t>(dsn::ranger::access_type_min);
+    int64_t act_max = static_cast<uint64_t>(dsn::ranger::access_type_max);
```

Comment:
better to use `uint64_t` or `auto`, since you have used `GetUint64` and `static_cast<uint64_t>`